mono/tests/Makefile.am

```diff
@@ -1046,7 +1046,6 @@ INTERP_DISABLED_TESTS = \
 	safehandle.2.exe \
 	stackframes-async.2.exe \
 	static-constructor.exe \
-	thread6.exe \
```

Comment:
Unrelated change.

Reply:
this commit probably fixes `thread6.exe`, so it's not really unrelated.